### PR TITLE
MINOR: add hint for setting an uncaught exception handler to JavaDocs

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

**Comment by mjsax** [comment]
Call for review @guozhangwang @bbejeck @dguy

**Comment by mjsax** [comment]
If we roll a new RC, this could go to `1.0.x`, too.

File: streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java [hunk 1]
```diff
@@ -723,6 +723,12 @@ private void checkBrokerVersionCompatibility() throws StreamsException {
      * Start the {@code KafkaStreams} instance by starting all its threads.
      * This function is expected to be called only once during the life cycle of the client.
      * <p>
+     * Because threads are started in the background this method does not block.
```
Comment (bbejeck) [inline streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java]: nit: background,

**Comment by bbejeck** [comment]
Thanks, @mjsax. Just one nit, otherwise LGTM.

**Comment by mjsax** [comment]
Updated this.

**Review by dguy — APPROVED** [review]
Thanks @mjsax, LGTM

**Comment by dguy** [comment]
merged to trunk and 1.0

**Pull request closed without merging**